airflow/models/baseoperator.py

```diff
@@ -467,11 +476,10 @@ class derived from this one results in the creation of a task object,
     :type doc_yaml: str
     """
 
-    # For derived classes to define which fields will get jinjaified
-    template_fields: Sequence[str] = ()
-    # Defines which files extensions to look for in the templated fields
-    template_ext: Sequence[str] = ()
-    # Template field renderers indicating type of the field, for example sql, json, bash
+    # Implementing Operator.
+    template_fields: Collection[str] = ()
```

Comment:
Why change from a collection to a sequence? (This might cause knock-on impacts for typing of all other operators)

Reply:
It’s the other way around, from Sequence to a Collection; Collection is looser than Sequence and should work in subclasses (I think; we can always change this back).